sdk/core/core/README.md

```diff
@@ -34,7 +40,7 @@ inline bool az_succeeded(az_result result) {
 
 Additional information could be passed using output parameters.
 
-## 2. Span of bytes
+### Span of bytes
 
 Defined in [inc/az_span.h](inc/az_span.h).
 
```

Comment:
These code snippets are out of date now.

Reply:
Once that is checked in we can cleanup these readmes.  This checkin makes them follow the standard format but does not address all inaccuracies.